src/components/common/admin/Header.tsx

```
@@ -0,0 +1,135 @@
+	showMenu?: boolean;
+	showSearchBar?: boolean;
+	showLanguage?: boolean;
+}
+
+interface MenuOption {
+	name: string;
+	icon?: React.ReactElement; // icon can be a React node
+	onClick?: () => void; // optional click handler
+}
+
+interface MenuItem {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'MenuItem'. Consider to delete it or rename it.

'MenuItem' is defined here:


(lint/suspicious/noRedeclare)